code/client/clrcore/External/BoneID.cs

```diff
@@ -27,27 +27,27 @@ public enum Bone
 		SKEL_Spine_Root = 57597,
 		SKEL_Spine0 = 23553,
 		SKEL_Spine1 = 24816,
-		SKEL_Spine2,
-		SKEL_Spine3,
+		SKEL_Spine2 = 24817,
+		SKEL_Spine3 = 24818,
 		SKEL_L_Clavicle = 64729,
 		SKEL_L_UpperArm = 45509,
 		SKEL_L_Forearm = 61163,
 		SKEL_L_Hand = 18905,
 		SKEL_L_Finger00 = 26610,
 		SKEL_L_Finger01 = 4089,
-		SKEL_L_Finger02,
+		SKEL_L_Finger02 = 64017,
```

Comment:
```suggestion
		SKEL_L_Finger02 = 4090,
```
The `64017` is for the right finger02 :)